wca-guidelines.md

```diff
@@ -170,6 +170,7 @@ To be more informative, each Guideline is classified using one of the following
 - A3c4+) [CLARIFICATION] If the use of a thin object inside the puzzle has been enforced, the judge must not remove the object on behalf of the competitor.
 - A3c4++) [CLARIFICATION] The competitor may remove the object during the solve, if they did not do it during the inspection (e.g. they forgot).
 - A5b+) [CLARIFICATION] While inspecting or solving the puzzle, the competitor may touch the puzzle with any part of their body. Exception: 3x3x3 One-Handed and 3x3x3 With Feet (see [Regulation C1b](regulations:regulation:C1b) and [Regulation D1b](regulations:regulation:D1b)).
+- A5b++) [CLARIFICATION] If a part of the competitor's puzzle pops out (see [Regulation 5a](regulations:regulation:5a)) and falls away, the competitor must not ask for assistance retrieving or locating the part and the judge should attempt to prevent others from assisting. If someone accidentally picks up a popped part for the competitor, the judge or competitor may ask them to place it on solving station and not to assist any further.
```

Comment:
"...to place it on *the* solving station ...", maybe?

Reply:
```suggestion
- A5b++) [CLARIFICATION] If a part of the competitor's puzzle pops out (see [Regulation 5a](regulations:regulation:5a)) and falls away, the competitor must not ask for assistance retrieving or locating the part and the judge should attempt to prevent others from assisting. If someone accidentally picks up a popped part for the competitor, the judge or competitor may ask them to place it on the solving station and not to assist any further.
```